sqldelight-compiler/src/main/kotlin/app/cash/sqldelight/core/lang/util/Arguments.kt

```diff
@@ -68,6 +70,14 @@ internal fun SqlExpr.inferredType(): IntermediateType {
       }
     }
 
+    is SqlMultiColumnExpression -> {
+      val idx = parentRule.exprList.indexOf(this)
+      val parentParent = parentRule.parent as SqlMultiColumnExpr
+
+      // The first multiColumnExpression is the column list
+      parentParent.multiColumnExpressionList[0].exprList[idx].type()
```

Comment:
Can I assume this will not return null?

From https://github.com/AlecStrong/sql-psi/blob/master/core/src/main/kotlin/com/alecstrong/sql/psi/core/sql.bnf#L237

these two expressions are not optional.

Reply:
Yes